EssentialFeed/EssentialFeediOSTests/Image Comments UI/ImageCommentsSnapshotTests.swift

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeediOS
+@testable import EssentialFeed
+
+class ImageCommentsSnapshotTests: XCTestCase {
+	func test_listWithComment() {
+		let sut = makeSUT()
+
+		sut.display(comments())
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "IMAGE_COMMENTS_light")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "IMAGE_COMMENTS_dark")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light, contentSize: .extraExtraExtraLarge)), named: "IMAGE_COMMENTS_light_extraExtraExtraLarge")
+	}
+
+	// MARK: - Helpers
+
+	private func makeSUT(file: StaticString = #file, line: UInt = #line) -> ListViewController {
+		let bundle = Bundle(for: ListViewController.self)
+		let storyboard = UIStoryboard(name: "ImageComments", bundle: bundle)
+		let controller = storyboard.instantiateInitialViewController() as! ListViewController
+		controller.loadViewIfNeeded()
+		controller.tableView.showsVerticalScrollIndicator = false
+		controller.tableView.showsVerticalScrollIndicator = false
```

Comment:
I believe one of this lines should hide the horizontal scroll indicator, is that right?
```suggestion
		controller.tableView.showsVerticalScrollIndicator = false
		controller.tableView.showsHorizontalScrollIndicator = false
```
Otherwise, remove the duplicate line.

Reply:
Yeah, right. That's my bad😅